clang/tools/libclang/CIndexCXX.cpp

```diff
@@ -27,6 +27,33 @@ unsigned clang_isVirtualBase(CXCursor C) {
   return B->isVirtual();
 }
 
+unsigned clang_visitCXXBaseClasses(CXType PT, CXFieldVisitor visitor,
+                                   CXClientData client_data) {
+  CXCursor PC = clang_getTypeDeclaration(PT);
+  if (clang_isInvalid(PC.kind))
+    return false;
+  const CXXRecordDecl *RD =
+      dyn_cast_or_null<CXXRecordDecl>(cxcursor::getCursorDecl(PC));
```

Comment:
are we supposed to be using the `if_present` versions or not?

Reply:
Yup, the `or_null` variants are deprecated in favor of the `is_present` ones.